src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppEcsFargate/README.md

```diff
@@ -1,14 +1,55 @@
-# Welcome to your CDK C# project!
+# AWS deploy tool deployment project
 
-This is a blank project for C# development with CDK.
+This .NET project is a deployment project used by the AWS deploy tool to deploy .NET applications to AWS. The project is made
+up of 2 parts.
 
-The `cdk.json` file tells the CDK Toolkit how to execute your app.
+First is a *.recipe file which defines all of the settings for deployment project. The recipe file is what
+the AWS.Deploy.CLI tool and the AWS Toolkit for Visual Studio use to drive the user experience to deploy a .NET application
+with this deployment project.
 
-It uses the [.NET Core CLI](https://docs.microsoft.com/dotnet/articles/core/) to compile and execute your project.
+The second part of the deployment project is a .NET AWS CDK project which defines the AWS infrastructure that the
+.NET application will be deployed to.
 
-## Useful commands
+## What is CDK?
+ 
+The AWS Cloud Development Kit (CDK) is an open source software development framework to define your cloud application
+resources using familiar programming languages like C#. In CDK projects, constructs are instantiated for each of the
+AWS resources required. CDK projects are used to generate an AWS CloudFormation template to be used by the
+AWS CloudFormation service to create a Stack of all of the resources defined in a template.
+
+Visit the following link for more information on the AWS CDK:
+https://aws.amazon.com/cdk/
+
+## Should I use the CDK CLI?
+
+In a regular CDK project the CDK CLI, acquired from NPM, would be used to execute the CDK project. Because AWS deploy
+tool deployment projects are made of both a recipe and a CDK project you should not use the CDK CLI directly on
+the deployment project.
+
+The AWS deploy tool from either AWS.Deploy.CLI tool package or AWS Toolkit for Visual Studio
+should be used to drive the experience. The AWS deploy tool will take care of acquiring the CDK CLI and executing the
+CDK CLI passing in all of the settings gathered in the AWS deploy tool.
+
+## Can I modify the deployment project?
+
+When a deployment projects is saved the project can be customized by adding more CDK constructs or customizing the existing
+CDK constructs.
+
+The default folder structure puts the CDK constructs originally defined by the deployment recipe into a folder called
+"Generated". It is recommended to not directly modify these files and instead customize the settings via the
+AppStack.CustomizeCDKProps() method. This allows the AWS deploy tool to be able easily updated the generated code
```

Comment:
grammar: "This allows the AWS deploy tool to easily update the generated"

Reply:
Removed the "be able". I had it in because we haven't actually implemented the update features and I was being cautious in case we don't get around to that for awhile.